### PR TITLE
Todo List app

### PR DESCRIPTION
**Pull request by Miguelus373**
### What this PR does:
- Adds all the modules for the project
- Adds all dependencies
- Adds all styles 
- Adds all necessary functions

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import todo from './todo-factory';
 import form from './form-load';
```
Comment (christianotieno) [inline src/index.js]: ### Mandatory
- [x] The `webpack` is configured well to some extent. I do love that you went an extra mile to add the starting scripts but somehow forgot to add the `webpack-dev-server` dependency so it won't, run. Please fix this.

- [x] We have some commented out code in the `stylesheet` that we might have forgotten to clean up. Kindly remove them. It helps keep our codebase cleaner and the entire project look professional. 




### Optional

- [x] By disabling ESLint rules,  don't you think it rids you the opportunity to write much cleaner code and suppresses the potential warnings you’d be getting, thus missing out on the education that ESLint rules provide? 😟  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no-ignore-linters.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)).

- [x] How about we had a delete button for the project if a user intends to delete a project?

- [x] Lastly, could our forms used to create a project or a todo have validations in all fields? (consider this a stretch, thank you)

**Review by christianotieno — CHANGES_REQUESTED** [review]
### Changes Required ♻️ 
Hello @Miguelus373 and @chinweokwu 👋 

Congratulations on the hard work with this todo project. The code is modularised and has used the javascript factory concept to generate todos for their respective projects dynamically and neatly well. The use of es6+ syntax is commendable and makes the entire codebase to look neat and solid. Keep it up 👏 

We, however, do have som few issues to take care of before we proceed down below 👇 

### Happy coding 💻 

#### Christian Otieno (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle:  :traffic_light: 

Hi @Miguelus373 and  @chinweokwu, great job with this project :+1: The UI looks awesome :fireworks: However, we have a few issues we can address before proceeding :+1: Kindly check the comments below: 
- The npm command `npm run start` returns the error in the screenshot below. Kindly check the  files where the errors are raised and fix it
WEBp
- The edit functionality is missing from our project. Let us kindly address it
edi

Once these are addressed, we can  submit another code review request :smile: 
_HAPPY CODING_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by whiz25 — APPROVED** [review]
## Approved 👏🏿 
Well done. The app looks good and runs locally. Feature for editing a todo item is also added as requested.
Merge and proceed.


### Martin Afani

**Pull request merged**